### PR TITLE
Clarify setup process for new users

### PR DESCRIPTION
**Pull request by haihala**
Previous README was misleading and didn't mention the need to register components.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,18 +17,41 @@
...
 
+## Usage
 
-More examples (with pictures) can be found in the [`examples folder`](examples).
+In order for custom components to show up in the inspector, you have to:
+
+1. Add an `InspectableRegistry` to the world as a resource
+2. Register the component to the `InspectableRegistry`
+3. `#[derive(Inspectable)]` for the component
+4. Add an inspector plugin
```
Comment (jakobhellermann) [inline README.md:31]: If you add the `WorldInspectorPlugin` first, you don't have to add an `InspectableRegistry`, you can just add to the existing one. I think the order would be either
- only display a single struct? Add an `InspectorPlugin<T>`
- want to use the WorldInspector?
1. add the `WorldInspectorPlugin`
2. `#[derive(Inspectable)]`
3. call `.register::<T>()` on the `InspectbleRegistry`.

File: README.md [hunk 2]
```diff
@@ -17,18 +17,41 @@
...
+use bevy::prelude::*;
+use bevy_inspector_egui::InspectableRegistry;
+
+let mut registry = app.insert_resource(InspectableRegistry::default())
+                      .world_mut()
+                      .get_resource_mut::<InspectableRegistry>()
+                      .expect("InspectableRegistry not initiated");
+
+registry.register::<Data>();
+registry.registre::<OtherComponent>();
```
Comment (jakobhellermann) [inline README.md:48]:
```suggestion
registry.register::<OtherComponent>();
```

**Review by jakobhellermann — CHANGES_REQUESTED** [review]
Thanks for the PR! One thing I'd like to change is that the readme now has he `InspectableRegistry` section, `Component`, `Inspector` and then the `InspectorPlugin` example.

Could you move the `InspectorPlugin` example to the top to the `Component` section and move `InspectableRegistry` below what is currently called `Inspector` and rename that to `World Inspector`?

File: README.md (hunk 1 above)
Comment (haihala) [reply README.md]: Is that better?

Comment (jakobhellermann) [reply README.md]: Yup thats good

**Review by jakobhellermann — APPROVED** [review]

**Pull request merged**